docs/building/MlNetMklDeps/MlNetMklDeps.nuspec

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/01/nuspec.xsd">
+  <metadata>
+    <id>MlNetMklDeps</id>
+    <version>0.0.0.6</version>
+    <title>This NuGet package provides Intel(R) MKL dependencies for ML.Net</title>
+    <authors>Intel Corporation</authors>
+    <owners>Microsoft</owners>
+    <projectUrl>https://dot.net/ml</projectUrl>
+    <licenseUrl>https://software.intel.com/en-us/license/intel-simplified-software-license</licenseUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>
+      Microsoft.ML.Net, Intel(R) MKL Dependencies. 
```

Comment:
(nit) `.NET` should always be all-caps - here and below.

Also, I would remove the `.` between `Microsoft` and `ML`.  And maybe switch it around: `Intel(R) MKL Dependencies for Microsoft ML.NET.`